v2/oauth/multiple-frontends-with-a-single-backend.mdx

```diff
@@ -0,0 +1,288 @@
+---
+title: Multiple frontend domains with a common backend 
+hide_title: true
+---
+
+import BackendSDKTabs from "/src/components/tabs/BackendSDKTabs"
+import OAuthFrontendTabs from "/src/components/tabs/OAuthFrontendTabs"
+import OAuthBackendTabs from "/src/components/tabs/OAuthBackendTabs"
+import CoreInjector from "/src/components/coreInjector"
+import AppInfoForm from "/src/components/appInfoForm"
+import TabItem from '@theme/TabItem';
+
+# Multiple Frontend Domains Using a Single Backend
```

Comment:
The title here and the title in the sidebar are not consistent. Please make sure consistency for all pages

Reply:
resolved